apps/next_dev_portfolio/components/Projects.js

```diff
@@ -0,0 +1,62 @@
+import { DotSmall } from "./Dot";
+import Link from "next/link";
+
+export default function About() {
+  return (
+    <>
+      <div id="projects"></div>
+      <section className="max-w-4xl px-4 mx-auto mt-32 space-y-8 text-center lg:lg:mt-52">
+        <h2 className="text-4xl text-white">
+          Projects
+          <DotSmall />
+        </h2>
+        <div className="gap-8 xl:grid xl:grid-cols-2">
+          <div className="flex items-center h-full max-w-xl p-4 mx-auto my-2 overflow-hidden rounded shadow-lg bg-neutral-800">
+            <div className="px-6 py-4">
+              <div className="mb-2 text-xl font-bold">Project tittle</div>
```

Comment:
Project Title should probably use a heading element, not a div semantically, and the parent could be an <article> here